pkg/kubelet/rkt/mock_os/mockfileinfo.go

```diff
@@ -0,0 +1,91 @@
+// Automatically generated by MockGen. DO NOT EDIT!
```

Comment:
`mockgen os FileInfo > mock_fileinfo/mockfileinfo.go`, happy to check-in a comment or `go:generate` for it, but I didn't see other examples of go generate and I don't expect this to change often (ever) because it's a go1 interface, not a k8 interface.